gulpfile.js

```diff
@@ -10,13 +10,16 @@ var open = require('gulp-open')
 var through = require('through2')
 var slug = require('slug')
 var rename = require('gulp-rename')
+var html_replace = require('gulp-html-replace')
 
 var md_path = process.env.INIT_CWD //#the initial directory
 var gulpfile_path = process.cwd() //#gulpfile.js location
 
-gulp.task('default', ['clean'], function() {
-  
+var content
```

Comment:
I don't think is very safe to use content as a global variable. We can call `getContent()` when we need it. I won't mind calling it once in `get-filename` and once in `default`. Using it as global implies implicit dependency between `default` and `get-filename` which is not really necessary.